### PR TITLE
Add double tap handler and tap-to-zoom test case. JB#56765 OMP#JOLLA-607

### PR DESCRIPTION
**Pull request by llewelld**

### CODE REVIEW

File: tests/manual/testdblick.html [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+            resetAnim("single");
+        }
+        function singleCheck() {
+            resetAnim("outer");
+        }
+    </script>
+    </head>
+
+    <body>
+        <p> Tap-to-zoom will only work if user-scalable is enabled.</p>
```
Comment (okodron) [inline tests/manual/testdblick.html:105]: Have you intentionally placed a space before the sentence?

**Review by okodron — APPROVED** [review]

File: tests/manual/testdblick.html (hunk 1 above)
Comment (llewelld) [reply tests/manual/testdblick.html]: No, no particular reason. I've removed it.

**Review by rainemak — APPROVED** [review]

**Comment by rainemak** [comment]
Rebased before merging.

**Comment by llewelld** [comment]
Rebased up to latest on next.

**Pull request merged**